### ISSUE TITLE
Output subdir ends up on .cue after merge

### ISSUE THREAD

**Issue by MediaMaquina** [putnam/binmerge#16]
I admit this might be me not being good at batch.
I do this:
merge.bat
```
for %%A in ("%~1") do set "InputFileName=%%~nA"
binmerge.exe "%~1" "merged\%InputFileName%"
```
The problem is the .cue file ends up picking up the "merged" directory like so:
`FILE "merged\file.bin" BINARY`
When it should just be:
`FILE "file.bin" BINARY`

ANything i can do in batch to fix this or it's just how the .exe works?

**Comment by putnam** [comment]
This was definitely a bug -- it is now fixed, see #12

**Comment by MediaMaquina** [comment]
Hooray 🎉 ty
